atlassian_jwt_auth/contrib/tests/test_requests.py

```diff
@@ -5,7 +5,7 @@
 
 import atlassian_jwt_auth
 from atlassian_jwt_auth.tests import utils
-from atlassian_jwt_auth.contrib.requests import JWTAuth, create_jwt_auth
+from atlassian_jwt_auth.contrib.requests_auth import JWTAuth, create_jwt_auth
```

Comment:
Yea, so this was a major mistake on my part originally. Having the module named `requests` blocks sibling modules from using `import requests` to import the real `requests` module.

It is a breaking API change but I think its an intelligent one.